frontend/src/players/players-list/players-list.tsx

```diff
@@ -96,43 +95,38 @@ export const PlayersList = ({ team, onFormSubmit }: Props): JSX.Element => {
   }, [team.players]);
 
   const deletePlayer = (playerId: number) => {
-    api.deletePlayer(team.id, playerId);
-    setPlayers(players.filter((player) => player.id !== playerId));
+    api.deletePlayer(team.id, playerId).then(() => {
```

Comment:
Requests to the API are best placed where this data is retrieved. In this case, the deletePlayer method would ideally be passed as a props from the parent